modules/ROOT/pages/distributed-acid-transactions-from-the-sdk.adoc

```diff
@@ -43,11 +52,24 @@ $ sudo yum install boost-devel
 
 [source,console]
 ----
-$ wget https://sdk-snapshots.couchbase.com/clients/transactions-cxx/couchbase-transactions-1.0.0-0.beta.1.63.el8.x86_64.tar
-$ tar xf couchbase-transactions-1.0.0-0.beta.1.63.el8.x86_64.tar
+$ wget https://packages.couchbase.com/clients/transactions-cxx/couchbase-transactions-1.0.0-1.253.el8.x86_64.tar
+$ tar xf couchbase-transactions-1.0.0-1.253.el8.x86_64.tar
```

Comment:
I didn't notice this earlier, but I thought we talked about not having that build number in there earlier.  Not going to worry about it for now, but maybe something we can look at later,